eng/pipelines/eng-common-sync.yml

```diff
@@ -0,0 +1,36 @@
+# Mirror the eng/common folder to all subscribed langauge repos.
+
+trigger:
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - eng/common
+
+pr: none
+
+resources:
+  repositories:
+    - repository: azure-sdk-build-tools
+      type: git
+      name: internal/azure-sdk-build-tools
+
+pool:
+  vmImage: windows-2019
```

Comment:
Why not a faster machine? Anything in here Windows specific? At least in the new template you added there isn't.

Reply:
+1, I'm under the impression that the Linux images have faster standup time. I'm not going to block approval or anything but if there's nothing windows specific I'd use the linux vm.

Reply:
Yeah I think this should run on linux I'll test that change.